lib/test_track_rails_client/rspec_helpers.rb

```diff
@@ -2,4 +2,8 @@
 
 RSpec.configure do |config|
   config.include TestTrackRailsClient::AssignmentHelper
+
+  config.before(:each) do
```

Comment:
This might not be necessary at all if we fix the fuzzing stuff. I'm pretty sure that the fuzzing logic needs to read the yml file already. I would wait on making any changes to stuff at the rspec level until we figure out if we need it at all.